style.css

```diff
@@ -348,3 +342,98 @@ h2.section-title {
   line-height: 20px;
   color: #253858;
 }
+
+/* Contact Form */
+
+.contact-form_container {
+  height: auto;
+  background-size: cover;
+  position: relative;
+}
+
+.background {
+  position: absolute;
+  background-image: url(Image/Background-Contact-Form.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  border-top-left-radius: 60px;
+  z-index: -11;
+  background-position: right;
+  width: 100%;
+  height: 100%;
+  background-color: #6070ff;
+}
+
+.Contact_Head-backgrond {
+  padding-top: 28px;
+  padding-left: 20px;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.info {
+  font-weight: bold;
+  font-size: 35px;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 16px;
+  margin-top: 120px;
+}
+
+.Text-description {
+  color: #ebebff;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  font-weight: 400px;
+  padding-right: 12px;
+  padding-top: 2px;
+  margin: auto;
+  font-family: Poppins, sans-serif;
+}
+
+.contact-form_container .form {
+  display: flex;
+  flex-direction: column;
+  max-width: 300px;
+  margin: 20px auto;
+  align-items: center;
+}
+
+.contact-form_container .form input,
+.textarea {
+  background: #fff;
+  padding: 10px 16px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  color: #172b4d;
+  width: 327px;
+  font-size: 17px;
+  margin: 10px 0;
+  font-family: Poppins, sans-serif;
+}
+
+.contact-button {
+  background-color: white;
+  padding: 12px 16px;
+  color: #7f8cff;
+  margin-right: 225px;
+  width: 125px;
+}
+
```

Comment:
- [ **OPTIONAL** ] Kindly make sure you also add the missing **Horizontal line** at the bottom of your page to stick to the Figma design.